app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -0,0 +1,343 @@
+import React, {
```

Comment:
There are a lot of components in this file. I think it'd be worth having a components folder with all the sub components and putting them altogether in this file